backend/event_processor/src/main/kotlin/com/feed_grabber/event_processor/email/EmailSender.kt

```diff
@@ -0,0 +1,45 @@
+package com.feed_grabber.event_processor.email
+
+import com.sendgrid.Method
+import com.sendgrid.Request
+import com.sendgrid.SendGrid
+import com.sendgrid.helpers.mail.Mail
+import com.sendgrid.helpers.mail.objects.Email
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.stereotype.Service
+import java.io.IOException
+
+@Service
+class EmailSender(
+        @Autowired val mailContentBuilder: MailContentBuilder,
+        @Value("\${send-grid-api-key}") val sendGridApiKey: String,
+        @Value("\${official-email}") val fromEmail: String
```

Comment:
These parameters specified in application.properties file. Of course, I can't add an API key directly to the properties file.
If someone will need this key then write to me and I'll provide it.